Justfile

```diff
@@ -159,6 +159,15 @@ check-lychee:
     @echo {{ if env("GITHUB_TOKEN", "") != "" { "" } \
              else { "Note: 'Too Many Requests' errors are allowed here but not in CI, set GITHUB_TOKEN to check them" } }}
 
+# check tools/protocol-schema-check/res/protocol_schema.toml
+check-protocol-schema:
+    rustup toolchain install nightly
+    rustup target add wasm32-unknown-unknown --toolchain nightly
```

Comment:
And in turn if we can remove the feature flag, we will also be able to remove the need for a nightly compiler here